chiperific/liters_tracker
# Add/edit form for stories

Test steps:

1. Make sure you have production data/run the migrations
2. Navigate to the monthly view
  a. Go to your homepage.
  b. Click on the "View the Data" tab
  c. Click on the "By Month" tab
  d. Load the September 2019 data
3. Use the Add button to add some stories (please try various combinations of fields left empty/filled!)
4. On save, make sure it redirects you back to the appropriate month's view.
5. Use the Edit button to edit some existing stories you added and validate the changes saved.
6. On save, make sure it redirects you back to the appropriate month's view.
